server/src/tests/ws.rs

```diff
@@ -745,3 +761,12 @@ async fn ws_server_backpressure_works() {
 	assert!(seen_backpressure_item);
 	assert!(seen_item_after_backpressure);
 }
+
+#[tokio::test]
+async fn notif_is_ignored() {
+	init_logger();
+	let addr = server().await;
+	let mut client = WebSocketTestClient::new(addr).with_default_timeout().await.unwrap().unwrap();
+
+	assert!(client.send_request_text(r#"{"jsonrpc":"2.0","method":"bar"}"#).with_default_timeout().await.is_err());
```

Comment:
this is weird but should not be "answered" :)

Reply:
Maybe this is worth a quick comment about why it can be ignored since it looks like a method call that should be responded to?